app/code/community/Algolia/Algoliasearch/Model/Source/JobMethods.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+class Algolia_Algoliasearch_Model_Source_JobMethods
+{
+    protected $_methods = array(
+        'saveConfigurationToAlgolia' => 'Save Configuration',
+        'moveIndex' => 'Move Index',
+        'deleteObjects' => 'Object Deletion',
+        'rebuildStoreCategoryIndex' => 'Store Category Reindex',
+        'rebuildCategoryIndex' => 'Category Reindex',
+        'rebuildStoreProductIndex' => 'Store Product Reindex',
+        'rebuildProductIndex' => 'Product Reindex',
+        'rebuildStoreAdditionalSectionsIndex' => 'Store Additional Section Reindex',
+        'rebuildAdditionalSectionsIndex' => 'Additional Section Reindex',
+        'rebuildStoreSuggestionIndex' => 'Store Suggestion Reindex',
+        'rebuildSuggestionIndex' => 'Sugesstion Reindex',
+        'rebuildStorePageIndex' => 'Store Page Reindex',
+        'rebuildPageIndex' => 'Page Reindex',
```

Comment:
Looks like there are 2 more methods available in the M1 extension : 
moveProductsTmpIndex
saveSettings

Reply:
Added the 2 additional methods to the list. Thanks for catching that!